pymodbus/_logging.py

```diff
@@ -0,0 +1,19 @@
+__all__ = ["apply_pymodbus_logging_config"]
+
+
+import logging
+from logging import NullHandler
+
+
+# ---------------------------------------------------------------------------#
+#  Block unhandled logging
+# ---------------------------------------------------------------------------#
+logging.getLogger(__name__).addHandler(NullHandler())
+
+
+def apply_pymodbus_logging_config():
```

Comment:
please change function name as well as file name.

Why not include the function directly in __init__ ?